applications/portal/backend/api/models/population.py

```diff
@@ -106,11 +106,11 @@ def _has_file_changed(self):
             and self.cells.file.name != current.cells.file.name
         )
 
-    def generate_cells(self, data_filepath: str):
+    def generate_cells(self, data_filepath: str, is_fiducial: bool = False):
```

Comment:
@afonsobspinto `is_fiducial` shouldn't this be an attribute of the population? I mean now we can generate cells (same population) with `is_fiducial` True and False. Not sure if that's meant like that